tests/rules/property.js

```diff
@@ -111,7 +111,19 @@ eslintTester.run("property", rule, {
         {
             code: "w.innerHTML = `<span>${'lulz'+'meh'}</span>`;",
             parserOptions: { ecmaVersion: 6 }
-        }
+        },
+
+        // testing unwrapSafeHTML spread
+        {
+            code: "this.imeList.innerHTML = Sanitizer.unwrapSafeHTML(...listHtml);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+
+
+        // Native method (Check customize code doesn't include these)
+        {
+            code: "document.toString = evil;"
```

Comment:
nonononoo.... nothing to do with that.

The config object parsing when looping though it defines a toString check making it "malicious" when not intended to be.